sound-equipment-fault-detection/webui/config

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lJRi9GNEFMeXJUWTB3RFFZSktvWklodmNOQVFFTEJRQXdGVEVUTUJFR0ExVUUKQXhNS2EzVmlaWEp1WlhSbGN6QWVGdzB5TkRBNE1qY3hOekUzTXpWYUZ3MHpOREE0TWpVeE56SXlNelZhTUJVeApFekFSQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLCkFvSUJBUURQN3EzY1NRNGJTNlpiR1IxYk5VUnViRlNzUEdMYlFLcWFPcmx6cDZtcW5hRFFLc1owWVFwY25BMTgKR3RzQ3g3YWI0WFhVUG5RcktPVFVhWlNHRmJDdS9TYTZVemNnaWVQTkg5REhQWEQvMit5YURldHZVTjV2KzI4dAo3bmNlWlBYclpiNFZGNmVyaHFVblB2L2NQWmo0TGw1STQra0lDUWs4NWdqQmJydXhwSlVhRldFQWZTTldFUDh3ClBzYTIyNFZoR042MktVcUpROW9yK0luZmJTOElXWVdrTytEb0RCTjJhcDZqaW9MczFWOEFHSmNSejdiam51K3AKMEZXQWVGcjlTTVVMaGJxNW9aVi9rR2w2N1dudDEvYVVqZHpsVHQ3QkJJd2k3NW9yL0F1MlkvamUrTTJtMjhYNQo5NmZaWFNRcmRsT3ZSM2ZOa3I1WUs1eWhjd1d4QWdNQkFBR2pXVEJYTUE0R0ExVWREd0VCL3dRRUF3SUNwREFQCkJnTlZIUk1CQWY4RUJUQURBUUgvTUIwR0ExVWREZ1FXQkJUNDRsUzh6SWZFeVE2RDFieEVaSmhTN04vaWpEQVYKQmdOVkhSRUVEakFNZ2dwcmRXSmxjbTVsZEdWek1BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQUNkVXdRUVNLVgpNdllzaTNOQnRwRVQ1Yml5bGFjYUJzZzVYeDRQSWhYTVJ1RittQXpQS2tNQUtjWmVRdktsNXA4ZWZHMnp5SC94CjRzZ0dBZUZUNm1tM3p5SDY0WDJHUTBqTStPVy90WlZwaW1oYm5TRjBhWE5GTEYxRjdpNEVIOWNMN29jYkplSksKYWM3eGVVQk5ORVhPR3daYWw2UEZROWx5NXk0UGxGZjdZN0YrKzJNVjZoMHBwQmhsMWdaUDAwNHFETjJFbkJ3Tgo3TktWS281RWZnQkJLc0M3L29uTjQvMlRXOU02RmRyMlpVcUhsbXUvNFlBUm90b0N2ZTdubXBObHJJdUNSR2JSCmNCbVcwVnhNODZqWDBlZlQ1Wnk0MVpUSDFTV3VSN3pBREl1YzNoYnQ4T3E3MWlDZ0JxSys1T1NDNGVOQnpHOXMKdTFDc1U0ajJlUU1VCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
Is this the secret key of k8s? It is recommended not to write the contents of the secret key directly, as there may be security issues.

Reply:
Thanks to that, I have used the InClusterConfig() function to get the Kubernetes cluster configuration instead of relying on the kubeconfig file.